src/main/java/salesforce/ui/pages/classic/contracts/NewContractPage.java

```diff
@@ -62,38 +63,46 @@ protected void waitForPageLoaded() {
     public void setAccountName(final String newAccountName) {
         this.accountName.sendKeys(newAccountName);
     }
+
     /**
      * Sets the CustomerSignedBy element.
      *
      * @param newCustomerSignedBy to be typed.
      */
-    public void setCustomerSignedBy(final WebElement newCustomerSignedBy) {
-        this.customerSignedBy = newCustomerSignedBy;
+    public void setCustomerSignedBy(final String newCustomerSignedBy) {
+        this.customerSignedBy.sendKeys(newCustomerSignedBy);
     }
+
     /**
      * Sets the CustomerSignedTittle element.
      *
      * @param newCustomerSignedTittle to be typed.
      */
-    public void setCustomerSignedTittle(final WebElement newCustomerSignedTittle) {
-        this.customerSignedTittle = newCustomerSignedTittle;
+    public void setCustomerSignedTittle(final String newCustomerSignedTittle) {
+        this.customerSignedTittle.sendKeys(newCustomerSignedTittle);
```

Comment:
You would use WebElementAction or something similar

Reply:
This class is for Classic version, Im going to improve it methods on the next PRs